misk-jobqueue-testing/src/main/kotlin/misk/jobqueue/FakeJobQueue.kt

```diff
@@ -0,0 +1,58 @@
+package misk.jobqueue
```

Comment:
It's not clear to me how to use this package. Could you add docs?

Reply:
done!